plugins/docker/docker_cpu

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/perl -w
+# -*- perl -*-
+
+=head1 NAME
+
+docker_cpu - Munin plugin to monitor docker container CPU usage.
+
+=head1 APPLICABLE SYSTEMS
+
+Should work on any Linux system that has docker support.
+
+=head1 CONFIGURATION
+
+Root privilege required to execute docker command.
+
+1. Create a new file named "docker" inside the folder /etc/munin/plugin-conf.d/
```

Comment:
This configuration file location is distribution specific.
It should be sufficient to just mention the required configuration.